src/extensions/extensionsHelper.ts

```diff
@@ -85,7 +91,12 @@ export const AUTOPOULATED_PARAM_PLACEHOLDERS = {
   DATABASE_INSTANCE: "project-id-default-rtdb",
   DATABASE_URL: "https://project-id-default-rtdb.firebaseio.com",
 };
-export type ReleaseStage = "stable" | "alpha" | "beta" | "rc";
+export const resourceTypeToNiceName: Record<string, string> = {
+  "firebaseextensions.v1beta.function": "Cloud Function",
+};
+export type ReleaseStage = "alpha" | "beta" | "rc" | "stable";
+const repoRegex = new RegExp("^https://github.com/[^/]+/[^/]+$");
```

Comment:
I think the `.`  in `github.com` needs to be escaped, otherwise it matches any single character

Reply:
Good catch!

Reply:
```suggestion
const repoRegex = new RegExp("^https://github\.com/[^/]+/[^/]+$");
```